Bottom: 278e7e8d8e54b2110cecdb57882bb072b8902e6e
Top:    d9f8204589927c5f6e03c5c6251342a8fe8b91ad
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-06 07:56:49 -0500

Refresh of add-vmdaedalus-inventory

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 6bd4fed8..590b9659 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -12,6 +12,7 @@
   - fetch:
       src: /tmp/ansible_variables
       dest: "{{inventory_hostname}}_ansible_variables"
+      flat: yes
 
 - name: Bacula
   hosts: bacula
